fs/artifact_fetcher.go

```diff
@@ -215,7 +217,10 @@ func FetchSociArtifacts(ctx context.Context, refspec reference.Spec, indexDesc o
 			if local {
 				return nil
 			}
-			return fetcher.Store(ctx, blob, rc)
+			if err := fetcher.Store(ctx, blob, rc); err != nil && !errors.Is(err, errdef.ErrAlreadyExists) {
+				return err
```

Comment:
nit `return fmt.Errorf("unable to store ztoc in local store: %w", err)`

Reply:
fix, PTAL @turan18